lib/app/resources/resourcefiles.go

```diff
@@ -343,6 +344,10 @@ func extractImages(objects []runtime.Object) (*ExtractedImages, error) {
 					containers = append(containers, job.Spec.Template.Spec.InitContainers...)
 				}
 			}
+		case *lensv1beta1.ImageSet:
+			for _, spec := range resource.Spec.Images {
```

Comment:
I'm not sure if I'm missing something, it's unclear to me how the registry field comes into play?

Reply:
So I was thinking the registry field will be useful when we add the actual admission server which will be looking at it and pulling images from the respective registry if it's specified. For users who just want to vendor additional images for now it's not really needed.

Or are you suggesting that if the registry field is specified, these images should be rewritten to point to this registry currently? We could do that but I don't think there is a real use-case for it at the moment, and the rewriting code will be going away soon anyway.

Reply:
Sorry, wasn't suggesting anything, I couldn't see how it was intended to be used, so I wasn't sure if this was for what we discussed in the design, whether we want the ability to pull from an alternate location, and whether the field registry was for that intended purpose.